features/conversation/conversation-message/conversation-message-content-types/conversation-message-reply.tsx

```diff
@@ -20,7 +20,7 @@ import {
 } from "@/features/conversation/conversation.store-context";
 import { usePreferredInboxName } from "@/hooks/usePreferredInboxName";
 import { captureError } from "@/utils/capture-error";
-import { DecodedMessageWithCodecsType } from "@/utils/xmtpRN/client.types";
+import { DecodedMessageWithCodecsType } from "@/utils/xmtpRN/client/client.types";
```

Comment:
I'm starting to think it might be a good idea if we have a very thin layer that wraps all of our messaging stuff before messaging stuff touches the application

The reason being we don't need all these complicated types littering our code base and we could just have a message that has content available on it with a content type that is a Concrete property that we can determine at that layer and so instead of these complicated decoded message with Kodex types, etc., we could simply have message conversation, etc.